contracts/token/BasicToken.sol

```diff
@@ -13,7 +13,15 @@ contract BasicToken is ERC20Basic, SafeMath {
 
   mapping(address => uint) balances;
 
-  function transfer(address _to, uint _value) {
+/*
+ * Fix for the ERC20 short address attack  
+ */
+  modifier onlyPayloadSize(uint size) {
+     assert(msg.data.length >= size + 4);
```

Comment:
The code is using Solidity v0.4.8, the assert function is implemented in SafeMath.sol, it is not the solidity one

Whenever Zeppelin migrates to v0.4.10 i agree that should be a require.